csharp/src/Google.Protobuf/Collections/RepeatedFieldExtensions.cs

```diff
@@ -0,0 +1,31 @@
+#region Copyright notice and license
+// Protocol Buffers - Google's data interchange format
+// Copyright 2015 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+#endregion
+
+using System;
+using Google.Protobuf.Collections;
+
+namespace Google.Protobuf
+{
+    /// <summary>
+    /// A set of extension methods on <see cref="RepeatedField{T}"/>
+    /// </summary>
+    public static class RepeatedFieldExtensions
+    {
+        /// <summary>Adds the elements of the specified span to the end of the <see cref="RepeatedField{T}"/>.</summary>
```

Comment:
```suggestion
        // Note: This method is an extension method to avoid ambigious overload conflict when the source is an array.
        /// <summary>Adds the elements of the specified span to the end of the <see cref="RepeatedField{T}"/>.</summary>
```